README.md

```diff
@@ -1,31 +1,69 @@
-# Reconstruction pipeline
-
-This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.
-It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB or used independently for modeling. 
+<h1 align="center">
+  <br>
+  <a href="http://www.amitmerchant.com/electron-markdownify"><img src="https://cdn.evbuc.com/eventlogos/50084740/tvbtaglinecrop.jpg" alt="The Virtual Brain" width="300"></a>
+  <br>
+  Reconstruction pipeline
+  <br>
+</h1>
+<h4 align="center">This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.</h4>
+<p align="center">
+  <a href="https://pegasus.isi.edu/downloads/">
+    <img src="https://img.shields.io/badge/Pegasus%20WMS-4.9.3-%2349A13D"
+         alt="Gitter">
+  </a>
+  <a href="https://neurostars.org/">
+	  <img src="https://img.shields.io/badge/Chat-on%20Neurostars-%23673B90">
+</a>
+  <a href="https://saythanks.io/">
+      <img src="https://img.shields.io/badge/SayThanks.io-%E2%98%BC-1EAEDB.svg">
+  </a>
+  <a href="https://www.paypal.com/">
```

Comment:
It is not really fitting for our dev model to have this link here for donations

Reply:
Okay, ma'am, I'll replace it with TVB download link (https://www.thevirtualbrain.org/tvb/zwei/brainsimulator-software). Please let me know If you want any other link to be displayed here.

Reply:
Also,ma'am, for the Say thanks button, you'll have to create an account in saythanks.io (https://saythanks.io/). Please let me know the link to your account, I'll link it to the button.

Reply:
If I think better, I would replace the SayThanks button with a link towards our Jira installation https://req.thevirtualbrain.org/

Reply:
Sure, ma'am. Any other button you would like to add?

Reply:
None comes to mind now. Thanks!

Reply:
Okay, all the changes are done. Please let me know if any other change is required.